### PR TITLE
[Universal Print] Adding support for $expand=documents on Get printJob

### PR DESCRIPTION
**Pull request by braedenp-msft**

### CODE REVIEW

File: api-reference/beta/api/printer-list-jobs.md [hunk 1]
```diff
@@ -103,6 +106,65 @@ Content-length: 461
...
+    {
+      "id": "5182",
+      "createdDateTime": "2020-02-04T00:00:00.0000000Z",
+      "createdBy": {},
+      "status": {
+        "processingState": "completed",
+        "processingStateDescription": "The print job has completed successfully and no further processing will take place."
+      },
+      "documents@odata.context": "https://graph.microsoft.com/beta/$metadata#print/printers('c05f3726-0d4b-4aa1-8fe9-2eb981bb26fb')/jobs('5182')/documents",
+      "documents": []
```
Comment (nilakhan) [inline api-reference/beta/api/printer-list-jobs.md:162]: please use an example with non-empty documents collection:

{
    "@odata.context": "https://graph.microsoft.com/beta/$metadata#print/printers('86b6d420-7e6b-4797-a05c-af4e56cd81bd')/jobs(documents())/$entity",
    "id": "31216",
    "createdDateTime": "2020-06-26T04:20:06.5715544Z",
    "createdBy": {
        "id": "{id}",
        "displayName": "{DisplayName}",
        "ipAddress": null,
        "userPrincipalName": "{userPrincipalName}",
        "oDataType": null
    },
    "status": {
        "processingState": "aborted",
        "processingStateDescription": "The print job has been aborted by a user or the printer and no further processing will take place."
    },
    "documents@odata.context": "https://graph.microsoft.com/beta/$metadata#print/printers('86b6d420-7e6b-4797-a05c-af4e56cd81bd')/jobs('31216')/documents",
    "documents": [
        {
            "id": "ca96c367-c3ad-478a-bbce-fbd1cd856e73",
            "displayName": "",
            "contentType": "application/oxps",
            "size": 276604,
            "configuration": {
                "quality": "medium",
                "dpi": 300,
                "feedDirection": null,
                "orientation": "landscape",
                "duplexMode": "oneSided",
                "copies": 2,
                "colorMode": "color",
                "inputBin": null,
                "outputBin": null,
                "mediaSize": null,
                "mediaType": null,
                "finishings": [],
                "pagesPerSheet": null,
                "multipageLayout": "clockwiseFromTopLeft",
                "collate": true,
                "scaling": null,
                "fitPdfToPage": null,
                "margin": null,
                "pageRanges": []
            }
        }
    ]
}

**Comment by braedenp-msft** [comment]
@Lauragra This PR is ready for your review as well!

Only two more coming your way this week. Promise 😋

**Comment by Lauragra** [comment]
@braedenp-msft , can you please look into this validation error:
Error: get_printjob_withdocumentdata:
      Failed to locate collection property 'value' in response.

**Comment by braedenp-msft** [comment]
> @braedenp-msft , can you please look into this validation error:
> Error: get_printjob_withdocumentdata:
> Failed to locate collection property 'value' in response.

Just pushed the fix 😊

**Pull request merged**